client/src/components/Overview/ImageExpandModal.jsx

```diff
@@ -15,15 +15,13 @@ export default function ImageExpandModal({
   const [dimensions, setDimensions] = useState({ width: 0, height: 0 });
 
   function handleMouseMove(event) {
-    console.log(event.target);
-    console.log(imageRef)
-    setMousePosition({ left: event.pageX, top: event.pageY });
-    const bgPosX = ((mousePosition.left / dimensions.width) * 100);
-    const bgPosY = ((mousePosition.top / dimensions.height) * 100);
+    setMousePosition({ left: event.clientX, top: event.clientY });
+    const bgPosX = (((mousePosition.left - 316) / dimensions.width) * 100);
+    const bgPosY = (((mousePosition.top - 84) / dimensions.height) * 100);
```

Comment:
Nice fix.  Potential fix: calculate position dynamically rather than hard coding numbers here.  What if styling changes in the future change the position?

Reply:
Good idea. I will work on that.